inertial_params:
    mass:
        m: 2.0
    MOI:
        Jxx: 0.020
        Jxy: 0
        Jxz: 0
        Jyy: 0.020
        Jyz: 0
        Jzz: 0.040
    CM:
        x: 0.00
        y: 0.001
        z: -0.01
aero_coeffs:
    C_T: 1.48e-7
    C_M: 2.524e-9
    l: 0.340